docs/os/windows/windows-vm41.md

```diff
@@ -0,0 +1,203 @@
+
+Create a VM named WindowsNew in [HVM](https://www.qubes-os.org/doc/hvm/) mode (Xen's current PVH limitations precludes from using PVH):
+
+~~~
+qvm-create --class StandaloneVM --label red --property virt_mode=hvm WindowsNew
+~~~
+
+Windows' installer requires a significant amount of memory or else the VM will crash with such errors:
+
+`/var/log/xen/console/hypervisor.log`:
+
+~~~
+p2m_pod_demand_populate: Dom120 out of PoD memory! (tot=102411 ents=921600 dom120)
+(XEN) domain_crash called from p2m-pod.c:1218
+(XEN) Domain 120 (vcpu#0) crashed on cpu#3:
+~~~
+
+So, increase the VM's memory to 4096MB (memory = maxmem because we don't use memory balancing).
+
+~~~
+qvm-prefs WindowsNew memory 4096
+qvm-prefs WindowsNew maxmem 4096
+~~~
+
+Disable direct boot so that the VM will go through the standard cdrom/HDD boot sequence:
+
+~~~
+qvm-prefs WindowsNew kernel ''
+~~~
+
+A typical Windows 7 installation requires between 25GB up to 60GB of disk space depending on the version (Home/Professional/...). Windows updates also end up using significant space. So, extend the root volume from the default 10GB to at least 50GB (note: it is straightforward to increase the root volume size after Windows is installed: simply extend the volume again in dom0 and then extend the system partition with Windows's disk manager).
+
+~~~
+qvm-volume extend WindowsNew:root 25g
+~~~
+
+The VM is now ready to be started; the best practice is to use an installation ISO [located in a VM](https://www.qubes-os.org/doc/standalone-and-hvm/#installing-an-os-in-an-hvm):
+
+~~~
+qvm-start --cdrom=untrusted:/home/user/windows_install.iso WindowsNew
+~~~
+
+Given the higher than usual memory requirements of Windows, you may get a `Not enough memory to start domain 'WindowsNew'` error. In that case try to shutdown unneeded VMs to free memory before starting the Windows VM.
+
+At this point you may open a tab in dom0 for debugging, in case something goes amiss:
+
+~~~
+tailf /var/log/qubes/vm-WindowsNew.log \
+   /var/log/xen/console/hypervisor.log \
+   /var/log/xen/console/guest-WindowsNew-dm.log
+~~~
+
+The VM will shutdown after the installer completes the extraction of Windows installation files. It's a good idea to clone the VM now (eg. `qvm-clone WindowsNew WindowsNewbkp1`). Then, (re)start the VM with `qvm-start WindowsNew`.
+
+The second part of Windows' installer should then be able to complete successfully.
+
+Finally, increase the VM's `qrexec_timeout`: in case you happen to get a BSOD or a similar crash in the VM, utilities like `chkdsk` won't complete on restart before `qrexec_timeout` automatically halts the VM. That can really put the VM in a totally unrecoverable state, whereas with higher `qrexec_timeout`, `chkdsk` or the appropriate utility has plenty of time to fix the VM. Note that Qubes Windows Tools also require a larger timeout to move the user profiles to the private volume the first time the VM reboots after the tools' installation.
+
```

Comment:
maybe add @jevank's comments about increasing this further ?

Reply:
I changed the value to 7200 (3 hours) after the last comments on Windows 10 updates taking hours to finish. Compared with that, QWT is harmless.